### PR TITLE
2104 feat/configure new user email

### PR DESCRIPTION
**Pull request by Ryoken**
#### What's this PR do?
Adds options to the organization settings that customizes the email sent out when a user is invited.

#### How should this be manually tested?
See screenshots.

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/2104

#### Screenshots (if appropriate)
Screen Shot 2021-02-16 at 2 37 50 PM

Screen Shot 2021-02-16 at 2 38 25 PM

Screen Shot 2021-02-16 at 2 38 44 PM

### CODE REVIEW

**Comment by RDmitchell** [comment]
@Ryoken -- so this is for someone in an existing organization inviting another person to their organization.

What about at the "superuser" level, ie, when adding an organization to an instance, having the ability to customize the email content?

**Comment by Ryoken** [comment]
> What about at the "superuser" level, ie, when adding an organization to an instance, having the ability to customize the email content?

Do you mean the email that reads like this?

> Hello {{ first_name }},
You are receiving this e-mail because you have been added to the '{{ new_org }}' organization by '{{ requested_by }}'. Please disregard this message if you are not expecting access to SEED.
You can view your new organization by logging into the SEED instance here:
{{ link }}
The SEED Team

If so, I can add settings to customize this email as well.

**Comment by Ryoken** [comment]
Added customization options for the other email:

Screen Shot 2021-02-18 at 10 24 07 AM

**Review by macintoshpie — CHANGES_REQUESTED** [review]
This looks good!

- I think we are still waiting for feedback regarding translation requirements, I just left a comment on the issue
- the only way to invite an existing user to an org on the frontend is through the /admin page, which requires superuser privileges. Since this page isn't accessible by non-superusers, I think we should git rid of the changes for customizing this email (this would be all of the `user_added_*` fields)

**Comment by RDmitchell** [comment]
@macintoshpie -- not sure I understand your 2nd bullet above, because the owner of an organiztion can invite others to become members of the org through the Organizations nav bar, Members link
image

**Comment by macintoshpie** [comment]
> @macintoshpie -- not sure I understand your 2nd bullet above, because the owner of an organiztion can invite others to become members of the org through the Organizations nav bar, Members link

@RDmitchell That link you reference treats it as if the email you provide doesn't already have an account on SEED (even if the email you provide is already linked to an existing SEED user).

There's another flow, from the admin page, where you can add _existing_ SEED users to another org, which sends a different email (the one that Ryo mentions here: https://github.com/SEED-platform/seed/pull/2590#issuecomment-781557843). My belief is that this latter one does not need to be customizable b/c of its inaccessibility to non-superusers.

**Comment by RDmitchell** [comment]
@macintoshpie -- right, the email coming from the superuser admin world doesn't need to be customized. It can stay as is.

File: seed/views/v3/organizations.py [hunk 1]
```diff
@@ -528,6 +536,62 @@ def warn_bad_units(kind, unit_string):
...
+
+        # update new user email subject option
+        new_user_email_subject = posted_org.get('new_user_email_subject')
+        if new_user_email_subject != org.new_user_email_subject:
+            org.new_user_email_subject = new_user_email_subject
+        if not org.new_user_email_subject:
+            org.new_user_email_subject = "New SEED account"
+
+        # update new user email content option
+        new_user_email_content = posted_org.get('new_user_email_content')
```
[the remark this answers is not in the document]
Comment (Ryoken) [reply seed/views/v3/organizations.py:554]: This will now be appended if it's not found in the body.  Good idea!

**Review by macintoshpie — APPROVED** [review]
Sliiick, very cool feature ⚡

**Pull request merged**